worlds/two_icubs_standup_world/README.md

```diff
@@ -0,0 +1,26 @@
+## To load two icubs standup world in Gazebo
+
+Normally, for loading a `world`, Gazebo needs to be launched in the folder where the `world` is. You can avoid this by appending these lines to your `.bashrc`:
+
+```
+source /usr/share/gazebo/setup.sh
+export GAZEBO_RESOURCE_PATH=$GAZEBO_RESOURCE_PATH:/PATH/WHERE/YOUR/WORLD/IS
+
+```
```

Comment:
This is good if you manually download the repo and you want to use the "source" world.  For usage within the superbuild, the world are installed in `<prefix>/gazebo/worlds` . It could make sense to mention this.

Reply:
I wrote this part of the `README.md` accordingly to the one of other worlds (e.g. [icub_standup_world](https://github.com/robotology/icub-gazebo-wholebody/tree/devel/worlds/icub_standup_world)).
I know @gabrielenava was having some problem when using `GAZEBO_RESOURCE_PATH`, which is the reason why I think he added those comments in the `README`.

Reply:
Ahhhh, now I got it. Let's open an issue either in this repo or in the superbuild to discuss this. We can either install directly the `world` files in `gazebo/worlds` or continue to use the subdirectories. However I would try to: 
* Keep the hierarchy consistent between the `worlds` directory in the repo and the `gazebo/worlds` installed directory, so adding `icub-gazebo-wholebody/worlds` or `<prefix>/gazebo/worlds` to `GAZEBO_RESOURCE_PATH` would have the same effect 
* Make sure we document the syntax for launching gazebo world out of the box from the superbuild, regardless of whether it is `gazebo worldName/worldName.world` or it is `gazebo  worldName.world`. In particular, this is a great check to add to a "Check your installation" in https://github.com/robotology/robotology-superbuild#dynamics .

Reply:
Issue created https://github.com/robotology/icub-gazebo-wholebody/issues/24 I agree on keeping the folders hierarchy in the installation.

Reply:
Ok.